# -*- coding: utf-8 -*-
# Generated by Django 1.11.3 on 2017-08-03 18:52
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('corebase', '0004_person'),
    ]

    operations = [
        migrations.CreateModel(
            name='PersonLogin',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('arrived_time', models.DateTimeField(auto_now_add=True)),
                ('public_certificate', models.TextField(help_text='Certificado público  de firma, para firma digital avanzada')),
                ('code', models.CharField(help_text='Código de verificación, es la identificación de la persona firmada', max_length=100)),
                ('person', models.CharField(help_text='Identificación de la persona solicitante', max_length=50)),
                ('data_hash', models.CharField(help_text='Suma hash de datos de tamaño máximo 130 caracteres, usando el\n                                 algoritmo especificado ', max_length=130)),
                ('algorithm', models.CharField(choices=[('sha256', 'sha256'), ('sha384', 'sha384'), ('sha512', 'sha512')], help_text=' Debe ser alguno de los siguientes: sha256, sha384, sha512', max_length=7)),
            ],
        ),
        migrations.AddField(
            model_name='person',
            name='expiration_datetime_token',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='person',
            name='last_error_code',
            field=models.SmallIntegerField(choices=[(1, 'Transacción satisfactoria'), (2, 'Error, persona no existe'), (3, 'Error no determinado')], default=1),
        ),
        migrations.AddField(
            model_name='person',
            name='token',
            field=models.CharField(blank=True, max_length=200, null=True),
        ),
    ]
